test/unit/package_set_test.rb

```diff
@@ -5,8 +5,16 @@
 
 module Packwerk
   class PackageSetTest < Minitest::Test
+    include RailsApplicationFixtureHelper
+
     setup do
-      @package_set = PackageSet.load_all_from("test/fixtures/skeleton/")
```

Comment:
I would like packwerk tests to treat `test/fixtures/skeleton` and `test/fixtures/minimal` as templates to copy into a temporary artifact instead of reading directly from them.

Reply:
Hmmm, I'm not sure if this would make things harder to understand, because we are mixing absolute and relative references to the application skeleon in the test. Can't we pass the app_dir to PackageSet?

Reply:
I just discovered that real calls to `PackageSet.load_all_from` pass in an absolute path, whereas this test suite uses a relative one.  We should test it the way it's used, and pass in an absolute path.  I'll switch over to `app_dir` and see what happens.